Engine/DataFeeds/Enumerators/FastForwardEnumerator.cs

```diff
@@ -83,7 +83,8 @@ public bool MoveNext()
             }
 
             // we've exhausted the underlying enumerator, iterator completed
-            return false;
+            _current = _enumerator.Current;
+            return true;
```

Comment:
`FastForwardEnumerator` is only used in live, backtesting uses `FillForwardEnumerator`